sdk/core/core-rest-pipeline/src/nodeHttpClient.ts

```diff
@@ -88,7 +88,7 @@ class NodeHttpClient implements HttpClient {
     let abortListener: ((event: any) => void) | undefined;
     if (request.abortSignal) {
       if (request.abortSignal.aborted) {
-        throw new AbortError("The operation was aborted.");
+        throw new AbortError("The operation was aborted. Request has already been canceled");
```

Comment:
Can you fix this for `fetchHttpClient` and `xhrHttpClient` as well?

Reply:
Done